src/app/tests/suites/certification/Test_TC_TBRM_2_4.yaml

```diff
@@ -84,3 +84,9 @@ tests:
       command: GetPendingDatasetRequest
       response:
           error: UNSUPPORTED_ACCESS
+
+    - label: "Cleanup: Close the commissioning window"
```

Comment:
These steps are present in the test plan, yes?  If not, then a test plan issue is needed to add them there.

Reply:
Please see: https://github.com/CHIP-Specifications/chip-test-plans/pull/4686